tools-v2/README.md

```diff
@@ -1997,6 +1999,27 @@ Output:
 +--------------------------------------+--------------+---------+
 ```
 
+#### recover
+
+##### recover volume
+
+recover volume from recycleBin
+
+Usage:
+```shell
+curve bs recover volume --path /test/volumepath --user root 
+```
+
+Output:
+```
++---------+
+| RESULT  |
++---------+
+| success |
++---------+
```

Comment:
maybe we can show path in table.

Reply:
thanks for suggestion. done.
![7GQY~XDDM6P_TJNI3`H C6X](https://github.com/opencurve/curve/assets/110807411/42d06ad7-2ad4-44b7-9202-c7c1ea8bf537)